cockatrice/src/gameselector.h

```diff
@@ -23,6 +23,7 @@ private slots:
     void actCreate();
     void actJoin();
     void checkResponse(const Response &response);
+    void updateButtonChoices(const QModelIndex &);
```

Comment:
If you wanna make a Pr to fix all the styles you outlined here, feel free to and I'll approve it